Bynder/Api/Queries/MediaQuery.cs

```diff
@@ -42,6 +42,12 @@ public class MediaQuery
         public string Keyword { get; set; }
 
         /// <summary>
+        /// The type of the asset
+        /// </summary>
+        [APIField("type")]
+        public string Type { get; set; }
```

Comment:
Since this property can only have a set of values. I think it would be appropriate to create an enum having those values, 
enum AssetType
{
Document,
Image, 
Video,
Audio
}

And then have a converter to convert it to lowercase.

[APIField("type", Converter = typeof(LowercaseConverter))
public AssetType? Type { get; set; }